img/signals/colors.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+define("color_red", "#E02E45");
```

Comment:
The convention in PHP is usually that constants are written uppercase, so this should be `COLOR_RED`